inc/views/base/usercp/attachments.twig

```diff
@@ -46,7 +46,7 @@
             </li>
             <li class="list__item">
                 <span class="list__field">{{ lang.attachstats_spaceused }}</span>
-                <span class="list__value">{{ friendly_usage }} ({{ percent }})</span>
+                <span class="list__value">{{ friendly_usage }} {{ (percent) }}</span>
```

Comment:
I'm not sure that this change is correct - I believe the intention was to wrap the output in `()`, but this just wraps the variable and won't be shown in the output.

Reply:
The issue was that when percent is zero, empty brackets were shown.
This change will show `()` only when percent is NOT NULL ??

Reply:
Ah, ok that makes sense.